webextensions/_locales/fr/messages.json

```diff
@@ -133,250 +133,254 @@
+  "api_requestedPermissions_type_tabs":      { "message": "Accéder aux onglets du navigateur" },
+  "api_requestedPermissions_type_cookies":   { "message": "Détecter les conteneurs des onglets du navigateur" },
 
 
-  "context_reloadTree_label": { "message": "&Actualiser cette arborescence" },
-  "context_reloadDescendants_label": { "message": "Actualiser les enfants" },
-  "context_closeTree_label": { "message": "Fermer cette arborescence" },
-  "context_closeDescendants_label": { "message": "Fermer les enfants" },
-  "context_closeOthers_label": { "message": "Fermer tout sauf cette arborescence" },
-  "context_collapseTree_label": { "message": "Plier cette arborescence" },
-  "context_collapseAll_label": { "message": "Plier tout" },
-    "context_expandTree_label": { "message": "Déplier cette arborescence" },
-    "context_expandAll_label": { "message": "Déplier tout" },
-  "context_bookmarkTree_label": { "message": "Marquer cette arborescence…" },
-  "context_groupTabs_label":    { "message": "Créer un nouveau group depuis les onglets" },
-  "context_topLevel_prefix": { "message": "Élément au plus haut niveau: " },
+  "context_reloadTree_label": { "message": "Actualiser cet arbre" },
+  "context_reloadDescendants_label": { "message": "Actualiser les descendants" },
+  "context_closeTree_label": { "message": "Fermer cet arbre" },
+  "context_closeDescendants_label": { "message": "Fermer les descendants" },
+  "context_closeOthers_label": { "message": "Fermer les autres onglets sauf cet arbre" },
+  "context_collapseTree_label": { "message": "Replier cet arbre" },
+  "context_collapseAll_label": { "message": "Replier tout" },
+  "context_expandTree_label": { "message": "Déplier cet arbre" },
+  "context_expandAll_label": { "message": "Déplier tout" },
+  "context_bookmarkTree_label": { "message": "Ajouter cet arbre aux marque-pages…" },
+  "context_groupTabs_label":    { "message": "Créer un nouveau groupe depuis les onglets" },
+  "context_topLevel_prefix": { "message": "Élément au plus haut niveau : " },
 
-  "context_collapsed_label": { "message": "Plié (for testing of a checkbox type menu)" },
-  "context_pinnedTab_label": { "message": "Épinglé (for testing of a radio type menu)" },
-  "context_unpinnedTab_label": { "message": "Désépinglé (for testing of a radio type menu)" },
+  "context_collapsed_label": { "message": "Replié (pour tester un menu type case à cocher)" },
+  "context_pinnedTab_label": { "message": "Épinglé (pour tester un menu type bouton radio)" },
+  "context_unpinnedTab_label": { "message": "Désépinglé (pour tester un menu type bouton radio)" },
 
 
-  "config_title": { "message": "Tree Style Tab Options" },
+  "config_title": { "message": "Options de Tree Style Tab" },
 
   "config_appearance_caption": { "message": "Apparence" },
 
-  "config_sidebarPosition_caption": { "message": "Position du panneau des onglets:" },
+  "config_sidebarPosition_caption": { "message": "Style du contenu pour la position du panneau :" },
```

Comment:
Not very clear but I can't propose a better sentence

Reply:
«Style du contenu relatif à la position du panneau» maybe? I mean, the English string is not really clear either, it’s the kind of things that are hard to put into clear sentences…